### PR TITLE
switch update.sh/check_online_status() from ping to curl to make it p…

### PR DESCRIPTION
**Pull request by mitch-geht-ab**
…roxy ready

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -3,9 +3,9 @@
 ############## Begin Function Section ##############
 
 check_online_status() {
-  CHECK_ONLINE_IPS=(1.1.1.1 9.9.9.9 8.8.8.8)
-  for ip in "${CHECK_ONLINE_IPS[@]}"; do
-    if timeout 3 ping -c 1 ${ip} > /dev/null; then
+  CHECK_ONLINE_DOMAINS=('https://github.com')
```
Comment (DerLinkman) [inline update.sh]: Maybe add one or two domains here as well.

google.com and hub.docker.com i would say.

The rest looks good

**Review by DerLinkman — CHANGES_REQUESTED** [review]
See comment

File: update.sh (hunk 1 above)
Comment (mitch-geht-ab) [reply update.sh]: Had added only the used domains by intention.

Comment (FreddleSpl0it) [reply update.sh]: i would at least add hub.docker.com as we pulling images from there.

**Review by DerLinkman — APPROVED** [review]
Looks fine now!

**Pull request merged**